Scheduled Tests Failed

https://github.com/hmasdev/autogen-werewolf/actions/runs/14376227020

```
Run mypy werewolf
  mypy werewolf
  mypy tests
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.11.11/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.11/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.11/x64
    Python[2](https://github.com/hmasdev/autogen-werewolf/actions/runs/14376227020/job/40309211170#step:5:2)_ROOT_DIR: /opt/hostedtoolcache/Python/[3](https://github.com/hmasdev/autogen-werewolf/actions/runs/14376227020/job/40309211170#step:5:3).11.11/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.11/x6[4](https://github.com/hmasdev/autogen-werewolf/actions/runs/14376227020/job/40309211170#step:5:4)
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.11/x64/lib
werewolf/const.py:12: error: Enum members must be left unannotated  [misc]
werewolf/const.py:12: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:13: error: Enum members must be left unannotated  [misc]
werewolf/const.py:13: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:14: error: Enum members must be left unannotated  [misc]
werewolf/const.py:14: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:18: error: Enum members must be left unannotated  [misc]
werewolf/const.py:18: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:22: error: Enum members must be left unannotated  [misc]
werewolf/const.py:22: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:23: error: Enum members must be left unannotated  [misc]
werewolf/const.py:23: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:24: error: Enum members must be left unannotated  [misc]
werewolf/const.py:24: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:2[5](https://github.com/hmasdev/autogen-werewolf/actions/runs/14376227020/job/40309211170#step:5:5): error: Enum members must be left unannotated  [misc]
werewolf/const.py:25: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:29: error: Enum members must be left unannotated  [misc]
werewolf/const.py:2[9](https://github.com/hmasdev/autogen-werewolf/actions/runs/14376227020/job/40309211170#step:5:9): note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:30: error: Enum members must be left unannotated  [misc]
werewolf/const.py:30: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:34: error: Enum members must be left unannotated  [misc]
werewolf/const.py:34: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:35: error: Enum members must be left unannotated  [misc]
werewolf/const.py:35: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:39: error: Enum members must be left unannotated  [misc]
werewolf/const.py:39: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:40: error: Enum members must be left unannotated  [misc]
werewolf/const.py:40: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:44: error: Enum members must be left unannotated  [misc]
werewolf/const.py:44: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:45: error: Enum members must be left unannotated  [misc]
werewolf/const.py:45: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:49: error: Enum members must be left unannotated  [misc]
werewolf/const.py:49: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:50: error: Enum members must be left unannotated  [misc]
werewolf/const.py:50: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
werewolf/const.py:51: error: Enum members must be left unannotated  [misc]
werewolf/const.py:51: note: See https://typing.readthedocs.io/en/latest/spec/enums.html#defining-members
Found [19](https://github.com/hmasdev/autogen-werewolf/actions/runs/14376227020/job/40309211170#step:5:20) errors in 1 file (checked 23 source files)
Error: Process completed with exit code 1.
```

--- a/werewolf/const.py
+++ b/werewolf/const.py
@@ -31,7 +31,7 @@ class ESide(Enum):
 
 
 class ESideVictoryCondition(Enum):
-    VillagersWinCondition = 'All werewolves are exclude from the game'
+    VillagersWinCondition = 'All werewolves are excluded from the game'
     WerewolvesWinCondition = 'Werewolves equal or outnumber half of the total number of players'  # noqa
 
 

--- a/werewolf/const.py
+++ b/werewolf/const.py
@@ -9,46 +9,46 @@ PREFIX_PLAYER_NAME: str = 'Player'
 
 
 class ESpeakerSelectionMethod(Enum):
-    auto: str = 'auto'
-    round_robin: str = 'round_robin'
-    random: str = 'random'
+    auto = 'auto'
+    round_robin = 'round_robin'
+    random = 'random'
 
 
 class EGameMaster(Enum):
-    Default: str = 'DefaultGameMaster'
+    Default = 'DefaultGameMaster'
 
 
 class ERole(Enum):
-    Villager: str = 'Villager'
-    Werewolf: str = 'Werewolf'
-    Knight: str = 'Knight'
-    FortuneTeller: str = 'FortuneTeller'
+    Villager = 'Villager'
+    Werewolf = 'Werewolf'
+    Knight = 'Knight'
+    FortuneTeller = 'FortuneTeller'
 
 
 class ESide(Enum):
-    Villager: str = 'Villager'
-    Werewolf: str = 'Werewolf'
+    Villager = 'Villager'
+    Werewolf = 'Werewolf'
 
 
 class ESideVictoryCondition(Enum):
-    VillagersWinCondition: str = 'All werewolves are exclude from the game'
-    WerewolvesWinCondition: str = 'Werewolves equal or outnumber half of the total number of players'  # noqa
+    VillagersWinCondition = 'All werewolves are exclude from the game'
+    WerewolvesWinCondition = 'Werewolves equal or outnumber half of the total number of players'  # noqa
 
 
 class EStatus(Enum):
-    Alive: str = 'Alive'
-    Excluded: str = 'Excluded'
+    Alive = 'Alive'
+    Excluded = 'Excluded'
 
 
 class EResult(Enum):
-    VillagersWin: str = 'VillagersWin'
-    WerewolvesWin: str = 'WerewolvesWin'
+    VillagersWin = 'VillagersWin'
+    WerewolvesWin = 'WerewolvesWin'
 
 
 class EChatService(Enum):
-    OpenAI: str = 'openai'
-    Google: str = 'google'
-    Groq: str = 'groq'
+    OpenAI = 'openai'
+    Google = 'google'
+    Groq = 'groq'
 
 
 MODEL_SERVICE_MAP: dict[str, EChatService] = {

